.github/workflows/lint.yml

```diff
@@ -0,0 +1,47 @@
+name: Linting
+
+on:
+  push:
+    paths:
+      - '**.py'
+  pull_request:
+    paths:
+      - '**.py'
+
+jobs:
+  Black:
```

Comment:
Other option is to have all three as separate steps in the same job? Less time spinning up runners then

Reply:
I'd done it this way to ensure all steps would run even if the previous ones failed. Found a workaround in the latest commit.